projects/manage-supervision-and-delius/src/test/kotlin/uk/gov/justice/digital/hmpps/service/OverviewServiceTest.kt

```diff
@@ -60,15 +60,19 @@ internal class OverviewServiceTest {
                     inBreach = false,
                     disposal = PersonGenerator.ACTIVE_ORDER,
                     eventNumber = "654321",
-                    mainOffence = PersonGenerator.MAIN_OFFENCE_1
```

Comment:
I know we already discussed adding these here, but may be worth just having some different test data specifically for the sentence controller, that way we can leave the overview alone. We can change later